www-client/forefix/forefix-128.1.0.0.recipe

```diff
@@ -0,0 +1,126 @@
+SUMMARY="Unofficial Haiku port of Mozilla Firefox"
+DESCRIPTION="Forefix is an open source web browser. \
+It achieves balance between ease of use and customization, catering to the \
+needs of both casual and power users."
+HOMEPAGE="https://github.com/kenz-gelsoft/gecko-dev"
+COPYRIGHT="1995-2024 Mozilla Developers and Contributors"
+LICENSE="MPL v2.0"
+REVISION="1"
+srcGitRev="8fb7f2c21739214acee7b8c2981b14c2a25bcb8e"
+SOURCE_URI="https://github.com/kenz-gelsoft/gecko-dev/archive/$srcGitRev.zip"
+CHECKSUM_SHA256="e95a3c7eca067b4edf57599e4bdfb5cd59e74042d715539bebac28234c00f573"
+SOURCE_DIR="gecko-dev-$srcGitRev"
+PATCHES="gecko-dev-$portVersion.patch"
+
+ARCHITECTURES="x86_64"
+
+PROVIDES="
+	forefix= $portVersion
+	app:Forefix= $portVersion
+	cmd:forefix= $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libatk_1.0 >= 0.23809.1
+	lib:libcairo >= 2.11800.0
+	lib:libcairo_gobject >= 2.11800.0
+	lib:libgdk_3 >= 3.2404.30
+	lib:libgdk_pixbuf_2.0 >= 0.4200.9
+	lib:libglib_2.0 >= 0.7800.0
+	lib:libgtk_3 >= 3.2404.30
+	lib:libharfbuzz >= 0.60830.0
+	lib:libintl >= 8.4.0
+	lib:libnspr4 >= 4.34.1
+	lib:libpango_1.0 >= 0.5400.0
+	lib:libpangocairo_1.0 >= 0.5400.0
+	lib:libpng16 >= 16.44.0
+	lib:libz >= 1.3.1
```

Comment:
Are these version constraints really required in the recipe? They are usually filled automatically by haikuporter.

Reply:
Thank you for feedback! I copy and paste  these version requirement from haikuporter command output. Is that not needed?

Reply:
They don't need to be specified explicitly in the recipe.

Reply:
I changed this in 8ae555ace2b4fb5995274716448ad09847ee9759 and indeed build succeeded.

Thanks!